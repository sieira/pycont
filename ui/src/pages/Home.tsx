import React from 'react'

const Home = (): JSX.Element => (
  <>
    <h1>You are in the home page</h1>
  </>
)

export default Home
