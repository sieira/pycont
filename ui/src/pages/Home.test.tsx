import React from 'react';
import { create } from 'react-test-renderer';
import { Provider } from 'react-redux';
import configureMockStore from 'redux-mock-store';
import thunkMiddleware from 'redux-thunk-recursion-detect';

import Home from './Home';

const mockStore = configureMockStore([thunkMiddleware]);
const store = mockStore({});

it('renders without crashing', () => {
  const homePage = create(<Provider store={store}><Home /></Provider>);
  expect(homePage.toJSON()).toMatchSnapshot();
});
