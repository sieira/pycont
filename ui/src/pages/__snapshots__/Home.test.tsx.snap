// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<h1>
  You are in the home page
</h1>
`;
